packages/ses/src/module-load.js

```diff
@@ -20,17 +21,46 @@ import {
   mapHas,
   mapSet,
   setAdd,
-  promiseCatch,
   promiseThen,
   values,
   weakmapGet,
+  iteratorNext,
+  iteratorThrow,
 } from './commons.js';
 import { assert } from './error/assert.js';
 
 const { Fail, details: d, quote: q } = assert;
 
 const noop = () => {};
 
+async function asyncTrampoline(generatorFunc, args, errorWrapper) {
+  // TODO: add iterator prototype methods to commons
```

Comment:
Noticed you did this.

```suggestion

```

Reply:
I remember removing this line.  :thinking: